### ISSUE TITLE
Accessibilty: Select dropdown or items not acknowledged by Orca screen reader

### ISSUE THREAD

**Issue by gedkirkham** [quasarframework/quasar#9304]
**Describe the bug**
As a user using [Orca](https://help.gnome.org/users/orca/stable/) screen reader to navigate Quasar select component, select component is not announced with an accessible relevant name upon focus. When user uses arrow key to traverse select option's list, option items are not read aloud by screen reader.

**Codepen/jsFiddle/Codesandbox (required)**
https://quasar.dev/vue-components/select#example--design-overview

**To Reproduce**
1. Activate Orca screen reader
2. Use tab key to focus select
3. Press enter/space to expand select dropdown
4. Press down arrow key to select initial select option

**Expected behavior**
1. Activate Orca screen reader
2. Use tab key to focus select
3. Screen reader reads out that select dropdown is in focus with an accessible relevant name
3. Press enter/space to expand select dropdown
4. Press down arrow key to select initial select option
5. First item in select option list is read aloud by screen reader

**Platform (please complete the following information):**
Quasar Version: 1.15.15

OS: Ubuntu
Browsers: Firefox, Chrome

**Comment by rstoenescu** [comment]
Improvement will be available in Quasar v1.15.23.
